wordpress.org/public_html/wp-content/themes/pub/wporg-support-2024/functions.php

```diff
@@ -5,6 +5,11 @@
  * @package WPBBP
  */
 
+/**
+ * Include locale specific styles.
+ */
+require_once dirname( dirname( get_template_directory() ) ) . '/wporg-parent-2021/inc/rosetta-styles.php';
```

Comment:
Is there a better way to reference this file?

We do something similar for the [theme.json](https://github.com/WordPress/wordpress.org/blob/trunk/wordpress.org/public_html/wp-content/themes/pub/wporg-support-2024/functions.php#L76)

Reply:
Maybe [get_theme_root](https://developer.wordpress.org/reference/functions/get_theme_root/)? I haven't tried it, but `get_theme_root( 'wporg-parent-2021' )` should work.

Reply:
Thanks, `require_once get_theme_root() . '/wporg-parent-2021/inc/rosetta-styles.php';` works 🎉